Build/Projects/MGCB.definition

```diff
@@ -10,6 +10,7 @@
     <Reference Include="System.Xml" />
     <Reference Include="MonoGame.Framework" />
     <Reference Include="MonoGame.Framework.Content.Pipeline" />
+    <Reference Include="MGCBReferences" />
```

Comment:
So is this the best way to configure this @hach-que so that the reference is only added for Windows and Mac?  Excluding Linux